tests/js/common/shell/shell-replicated-logs-cluster.js

```diff
@@ -152,7 +152,7 @@ function ReplicatedLogsWriteSuite () {
       let globalStatus = log.globalStatus();
       assertEqual(globalStatus.specification.source, "RemoteAgency");
       let status = log.status();
-      assertEqual(status, globalStatus);
+      assertEqual(Object.keys(status).sort(), Object.keys(globalStatus).sort());
```

Comment:
Can you maybe insert a comment that explains the rationale behind just comparing the keys?

Alternatively and more complicatedly write a function whose name conveys why you only compare keys and then write the rationale as a comment there. 

This is mainly for the benefit of future-us who will come across the code and wonder why.

Reply:
That's fair, I will leave a comment.